gradle/mockito-core/inline-mock.gradle

```diff
@@ -0,0 +1,13 @@
+task copyMockMethodDispatcher(type: Copy) {
+    from "${buildDir}/classes/main/org/mockito/internal/creation/bytebuddy"
+    into "${buildDir}/classes/main/org/mockito/internal/creation/bytebuddy"
+    include "MockMethodDispatcher.class"
+    rename { String fileName ->
+        fileName.replace('.class', '.raw')
+    }
+}
+classes.dependsOn(copyMockMethodDispatcher)
```

Comment:
Above change makes the compiled classes directory contain both: the 'raw' and normal class. Is this what we want?